bigchaindb/pipelines/block.py

```diff
@@ -44,6 +44,8 @@ def filter_tx(self, tx):
         if tx['assignee'] == self.bigchain.me:
             tx.pop('assignee')
             tx.pop('assignment_timestamp')
+            # required for mongodb
+            tx.pop('_id', None)
```

Comment:
Should this be handled by something on a `backend` level instead of here? I can help here if you want.